crates/sui-indexer-alt-framework/src/lib.rs

```diff
@@ -302,6 +302,11 @@ impl Indexer {
             );
         }
 
+        let cp_mapping_enabled = self.added_pipelines.contains("cp_mapping");
+        if !cp_mapping_enabled {
+            warn!("cp_mapping pipeline is not enabled on this indexer instance. Tables configured for pruning may not actually be pruned if cp_mapping is not written to.");
```

Comment:
I find this warning a bit aggressive, given that we would be running a dozen pods and only one will have this enabled. It also gives a confusing signal and make you wonder if you should actually enable cp_mapping on this pod.

Reply:
To limit the number of warning messages we emit, we could have indexer-alt determine whether it needs pruning, and set some flag on indexer from which the framework can decide whether to warn

But if every instance knows it needs pruning, then every instance would still end up with a warning. We could instead have an INFO level message during startup, or not have any logs whatsoever and rely on the docs and readme

Reply:
Why don't we output a warning from the pruner task if it tries to fetch the `cp_mapping` and it can't?